### ISSUE TITLE
Selectable optgroup ?

### ISSUE THREAD

**Issue by ncelerier** [crlcu/multiselect#139]
Hello, 

I'm using optgroups and I would like to be able to :
- click on an optgroup ;
- click on the button to move the whole optgroup (and its options) to the left or to the right. 

It looks like we can't do that out of the box, and I guess other people might need this feature too. So it might be great if this feature could be added. 

If this feature can't be added, I'm curious if any of you has already implemented this feature ? I don't really want to change the optgroup itself to an option because then I'll have to implement code so that when I move an optgroup option I'll have to copy its optgroup too. 

Thank you for your help.

**Comment by ncelerier** [comment]
It looks like it is working properly with this piece of code:

```
$(document).ready(function($) {                
    $("optgroup").on("click", function() { 
        // Select all options of the clicked optgroup
        $(this).children().prop("selected", "selected");
    });
                            
    $("optgroup option").on("click", function(event) {
        // When you click on an option inside an optgroup, all the options in this
        // optgroup are selected. We dont want this behaviour.
        event.stopPropagation();
     });

     $("[id^=optgroup]").multiselect();     
 }); 
```

**Comment by crlcu** [comment]
Hi @ncelerier 

I've just released the v2.4.0 which handles click-in on `optgroup`'s.
